users/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db       import models
+from core.models     import TimeStampedModel
+from products.models import Product
+
+class User(TimeStampedModel):
+    firstname = models.CharField(max_length=50)
+    lastname  = models.CharField(max_length=50)
+    email     = models.EmailField(max_length=50,unique=True)
+    password  = models.CharField(max_length=200)
+    friend    = models.ManyToManyField('self', through='Friend', symmetrical=False)
+    cart      = models.ManyToManyField(Product, through='Cart')
```

Comment:
ManyToMany 필드를 사용하시는 경우에는 속성명을 복수로 만들어주셔야 합니다. 
이후에 `user.friends.all()` 혹은 `user.carts.all()` 형태로 가독성 좋은 코드를 작성할 수 있습니다.
```suggestion
    friends    = models.ManyToManyField('self', through='Friend', symmetrical=False)
    carts      = models.ManyToManyField(Product, through='Cart')
```